packs/dimensiondata/actions/balancer_attach_member.py

```diff
@@ -10,10 +10,10 @@
 class BalancerAttachMemberAction(BaseAction):
     api_type = 'loadbalancer'
 
-    def run(self, region, member_id, balancer_id, member_ip, member_port):
+    def run(self, region, balancer_id, member_ip, member_port):
         driver = self._get_lb_driver(region)
         balancer = driver.get_balancer(balancer_id)
-        member = Member(id=member_id, ip=member_ip, port=member_port)
+        member = Member(id=None, ip=member_ip, port=member_port)
```

Comment:
I believe this diverged from the Libcloud API, but I'm fine with that since it's dimensiondata pack.